server/controllers/boards.js

```diff
@@ -117,4 +118,47 @@ exports.moveCard = asyncHandler(async (req, res) => {
   destCol.cards.splice(row, 0, cardId);
   await destCol.save();
   res.sendStatus(200);
+})
+
+exports.getDetails = asyncHandler(async (req, res) => {
+  const cardDetails = await CardDetail.find();
```

Comment:
cardDetails? I don't see any imports of cardDetail, can you explain what you are achieving here?

Reply:
I had a separate schema for card detail before and I will fix it soon

Reply:
Looks like Ajay is mentioning in the imports, I am not able to see that CardDetail being imported in this file?

Reply:
CardDetail was created and imported to board.js but I ended up merging two schemas and forgot to re-code according to the change.